.github/workflows/go.yml

```diff
@@ -35,12 +35,7 @@ jobs:
         if: steps.cache.outputs.cache-hit != 'true'
         run: go mod download
 
-      - name: Lint
+      - name: Test and lint
         run: |
           export PATH=$PATH:~/go/bin
-          ./bin/golangci-lint run --deadline=10m \
-            --disable-all --enable govet \
-            --enable staticcheck --enable gosimple \
-            --enable unconvert --enable ineffassign \
-            --enable revive --enable goimports \
-            --enable unparam
\ No newline at end of file
+          ./run_tests.sh
```

Comment:
This is causing CI failure. I think add `./bin` to PATH as well. Not sure if `~go/bin` is needed in path given this.

Reply:
Yeah, not sure why it doesn't get found.  Added explicit path.